test/e2e/nodepool_nto_machineconfig_test.go

```diff
@@ -100,6 +101,9 @@ func (mc *NTOMachineConfigRolloutTest) BuildNodePoolManifest(defaultNodepool hyp
 func (mc *NTOMachineConfigRolloutTest) Run(t *testing.T, nodePool hyperv1.NodePool, nodes []corev1.Node) {
 	ctx := mc.ctx
 
+	// Add initial delay to reduce concurrent API load with other tests
```

Comment:
I'm not sure how the Sleeps can guarantee anything determiniscally?
Let's drop them and keep adjusting the WithInterval